src/node.cc

```diff
@@ -1053,30 +1053,6 @@ void SetupProcessObject(Environment* env,
     READONLY_PROPERTY(process, "traceDeprecation", True(env->isolate()));
   }
 
-  // TODO(refack): move the following 4 to `node_config`
-  // --inspect-brk
-  if (env->options()->debug_options().wait_for_connect()) {
-    READONLY_DONT_ENUM_PROPERTY(process,
```

Comment:
Unfortunately we can't just remove these with deprecating them first

Reply:
These properties _breakFirstLine, _breakNodeFirstLine, _deprecatedDebugBrk and _invalidDebug are not used anywhere in the source. Should I put these back?